### PR TITLE
fix: it is now the mender-test-bot which triggers the nightlies

### PR DESCRIPTION
**Pull request by oleorhagen**
Ticket: None
Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Review by mzedel — COMMENTED** [review]
since we do the same in another location: https://github.com/mendersoftware/mantra/blob/master/ui/pages/nightlies.js#L73-L74 - is there anything against using the same approach here? maybe even the same querying method (so graphql)?

**Comment by oleorhagen** [comment]
> since we do the same in another location: https://github.com/mendersoftware/mantra/blob/master/ui/pages/nightlies.js#L73-L74 - is there anything against using the same approach here? maybe even the same querying method (so graphql)?

I would not be against it at least.

File: scripts/get-nightly-gitlab.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def iterate_dates(start_date, total_days, delta_days=-1):
 
 def iterate_nightlies(start_date, total_days):
     # Filter with user that triggers master and nightlies:
-    parameters_base = "username=lluiscampos"
+    parameters_base = "username=mender-test-bot"
```
Comment (lluiscampos) [inline scripts/get-nightly-gitlab.py:36]: Additionally,

With the intention of experimenting a bit with GCP resources, I yesterday moved the nightly to start at 10pm, so the filter few lines down is also off :see_no_evil:

Comment (oleorhagen) [reply scripts/get-nightly-gitlab.py:36]: Oh.. hah. Does it show as `updated` when it's finished? :thinking:

Comment (lluiscampos) [reply scripts/get-nightly-gitlab.py:36]: I don't know what you are asking...

In any case, I restored schedule to 1am so you can ignore my comment.

**Comment by oleorhagen** [comment]
Doing the GraphQL part separately

**Pull request merged**